### ISSUE TITLE
FieldAccessException on 1.17

### ISSUE THREAD

**Issue by GriffinCodes** [lucko/helper#106]
Not sure if this is a helper error or a ProtocolLib error

```
[19:43:35] [Server thread/WARN]: [ProtocolLib] Loaded class me.lucko.helper.scoreboard.PacketScoreboardTeam from helper v5.6.7 which is not a depend, softdepend or loadbefore of this plugin.
[19:43:35] [Server thread/WARN]: FieldAccessException: No field with type net.minecraft.network.chat.IChatBaseComponent exists in class PacketPlayOutScoreboardTeam.
[19:43:35] [Server thread/WARN]: 	at com.comphenix.protocol.reflect.StructureModifier.writeInternal(StructureModifier.java:365)
[19:43:35] [Server thread/WARN]: 	at com.comphenix.protocol.reflect.StructureModifier.write(StructureModifier.java:345)
[19:43:35] [Server thread/WARN]: 	at me.lucko.helper.scoreboard.PacketScoreboardTeam.newUpdatePacket(PacketScoreboardTeam.java:384)
[19:43:35] [Server thread/WARN]: 	at me.lucko.helper.scoreboard.PacketScoreboardTeam.setColor(PacketScoreboardTeam.java:277)
[19:43:35] [Server thread/WARN]: 	at me.pugabyte.nexus.features.minigames.models.scoreboards.MinigameScoreboard$Teams.lambda$getScoreboardTeam$0(MinigameScoreboard.java:93)
[19:43:35] [Server thread/WARN]: 	at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1224)
[19:43:35] [Server thread/WARN]: 	at me.pugabyte.nexus.features.minigames.models.scoreboards.MinigameScoreboard$Teams.getScoreboardTeam(MinigameScoreboard.java:91)
[19:43:35] [Server thread/WARN]: 	at me.pugabyte.nexus.features.minigames.models.scoreboards.MinigameScoreboard$Teams.lambda$update$2(MinigameScoreboard.java:111)
[19:43:35] [Server thread/WARN]: 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
[19:43:35] [Server thread/WARN]: 	at me.pugabyte.nexus.features.minigames.models.scoreboards.MinigameScoreboard$Teams.update(MinigameScoreboard.java:104)
[19:43:35] [Server thread/WARN]: 	at me.pugabyte.nexus.features.minigames.models.Match.start(Match.java:232)
[19:43:35] [Server thread/WARN]: 	at me.pugabyte.nexus.features.minigames.models.Lobby$LobbyTimer.lambda$start$1(Lobby.java:103)
[19:43:35] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftTask.run(CraftTask.java:100)
[19:43:35] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468)
[19:43:35] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1488)
[19:43:35] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:477)
[19:43:35] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1404)
[19:43:35] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1180)
[19:43:35] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
[19:43:35] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:831)
```

**Comment by PedroMPagani** [comment]
The author does not add support anymore(as far as I know) and please notice, 1.17 changed many things on the backend api, such as even just variable names breaking almost everything

**Comment by lucko** [comment]
Probably fixed by 068605dbb7a4df6247f5d8c9fe0e3ceedac6f69c
